src/components/scenes/ScanScene.js

```diff
@@ -45,6 +45,12 @@ export class Scan extends React.Component<Props> {
                 <T style={styles.bottomButtonText}>{s.strings.fragment_send_flash}</T>
               </View>
             </TouchableHighlight>
+            <TouchableHighlight style={styles.bottomButton} onPress={this._onToggleAddressModal} underlayColor={THEME.COLORS.SECONDARY}>
+              <View style={styles.bottomButtonTextWrap}>
+                <IonIcon style={styles.addressBookIcon} name="ios-book" size={scale(24)} />
+                <T style={styles.bottomButtonText}>{s.strings.fragment_send_address}</T>
```

Comment:
The text of the button should not say "Address", instead something like "Enter key" is better.